### PR TITLE
Cleanup: remove github sso fragments; some random details.

### PR DESCRIPTION
**Pull request by fisx**

### CODE REVIEW

**Comment by fisx** [comment]
@jkarni: same here.  (-:

File: thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs [hunk 1]
```diff
@@ -39,8 +36,6 @@ data ThentosA3Error =
     | A3BackendErrorResponse Int LBS
     | A3BackendInvalidJson String
     | SsoErrorUnknownCsrfToken
```
Comment (jkarni) [inline thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs]: `SsoErrorUnknownCsrfToken` <- should this be removed?

File: thentos-core/src/Thentos.hs [hunk 2]
```diff
@@ -71,8 +71,6 @@ main = makeMain $ \ actionState mBeConfig mFeConfig cmd ->
 
             void $ concurrently backend frontend
 
-        RunSso -> error "RunSso: not implemented."
```
Comment (jkarni) [inline thentos-core/src/Thentos.hs]: Can we just remove `cmd`/`Command` altogether? It doesn't seem to be serving a purpose anymore.

File: thentos-tests/src/Thentos/Test/Core.hs [hunk 3]
```diff
@@ -200,22 +198,13 @@ loginAsGod actionState = do
...
-
-
 -- * misc
 
 -- | Like 'Data.Aeson.decode' but allows all JSON values instead of just
 -- objects and arrays.
 --
--- Copied from https://github.com/haskell-servant/servant-client
--- (FIXME: also available from attoparsec these days.  replace!)
+-- Once we don't need snap any more we can upgrade to aeson >= 0.10 and use 'Aeson.decodeEither'
```
Comment (ChristianSi) [inline thentos-tests/src/Thentos/Test/Core.hs]: 'decodeEither' should read 'eitherDecode', I think (there is no decodeEither in aeson).

File: thentos-tests/src/Thentos/Test/Core.hs [hunk 4]
```diff
@@ -184,7 +181,8 @@ createActionState dbname config = do
     connPool <- createDb dbname
     return $ ActionState (connPool, rng, config)
 
--- | Create a connection to an empty DB.
+-- | Create a connection to an empty DB.  (NOTE: There is no need for a 'destroyDb', since 'Pool'
```
Comment (jkarni) [inline thentos-tests/src/Thentos/Test/Core.hs]: I'm generally not in favour of incorporating documentation from external libraries :). I'd also *imagine* `Pool` is sensibly-garbage collected, so it's not surprising behaviour.

File: thentos-tests/thentos-tests.cabal [hunk 5]
```diff
@@ -94,6 +94,7 @@ test-suite tests
   build-depends:
       aeson >=0.8.0.2 && <0.9
     , base >=4.8.1.0 && <5
+    , configifier >=0.0.5 && <0.1
```
Comment (jkarni) [inline thentos-tests/thentos-tests.cabal]: why do we now need this and `servant-server`? (And didn't before?)

File: thentos-tests/src/Thentos/Test/Core.hs (hunk 3 above)
Comment (jkarni) [inline thentos-tests/src/Thentos/Test/Core.hs]: also, it exists in 0.9, for what it's worth.

**Comment by fisx** [comment]
waiting for travis...

**Pull request merged**